Description: Ensure all errors are included from initialization
 Previously only Level 1 errors were added during init.
Origin: upstream, http://git.ghostscript.com/?p=ghostpdl.git;h=78ed0a8
Author: Chris Liddell <chris.liddell@artifex.com>
Forwarded: yes
Last-Update: 2018-10-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Resource/Init/gs_init.ps
+++ b/Resource/Init/gs_init.ps
@@ -1110,8 +1110,7 @@
    .setglobal
  } bind def
 ErrorNames
- { dup .registererror /VMerror eq {exit} if
- } forall
+ { .registererror} forall
 errordict begin
 % The handlers for interrupt and timeout are special; there is no
 % 'current object', so they push their own name.
